src/app/page.tsx

```
@@ -1,54 +1,241 @@
+			<section className={styles.faqSection}>
+				<div className={styles.faqContent}>
+					<h2 className={styles.faqTitle}>Frequently Asked Questions</h2>
+					<div className={styles.faqGrid}>
+						{faqs.map((faq, idx) => (
+							<div
+								key={idx}
+								className={styles.faqItem}
+								onClick={() => setOpenIndex(openIndex === idx ? null : idx)}
+								style={{ cursor: "pointer" }}
+							>
+								<h3 className={styles.faqQuestion}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)